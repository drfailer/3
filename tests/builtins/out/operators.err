[[1;33mWARN[0m]: /builtins/operators.:10: in assignment, [1;34ma[0m is of type [1;34mint[0m but the value assigned is of type [1;34mflt[0m.
[[1;33mWARN[0m]: /builtins/operators.:16: in assignment, [1;34mc[0m is of type [1;34mint[0m but the value assigned is of type [1;34mflt[0m.

