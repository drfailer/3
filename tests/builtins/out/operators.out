a: 3
b: 7.8
a: 19
a: -4
